tests/cephfs/multifs/multifs_kernelmounts.py

```diff
@@ -100,3 +100,4 @@ def run(ceph_cluster, **kw):
         client1.exec_command(
             sudo=True, cmd="cp /etc/fstab.backup /etc/fstab", check_ec=False
         )
+        client1.exec_command(sudo=True, cmd="rm -rf /etc/fstab.backup", check_ec=False)
```

Comment:
Would `mv` work instead of `cp` in the earlier line... this way we can avoid this line